### PR TITLE
Add warning when username contains uppercase

### PR DESCRIPTION
**Pull request by rkevin-arch**
@aalmanza1998 Add test cases?

### CODE REVIEW

**Comment by Lawrence37** [comment]
I think we should print the warnings in yellow.

**Comment by aalmanza1998** [comment]
> I think we should print the warnings in yellow.

Done

**Comment by rkevin-arch** [comment]
LGTM

File: ngshare_exchange/course_management.py [hunk 1]
```diff
@@ -238,7 +241,9 @@ def add_students(args):
             user = s['username']
             if s['success']:
                 prGreen(
-                    '{} was sucessfuly added to {}'.format(user, args.course_id)
+                    '{} was successfuly added to {}'.format(
```
Comment (Lawrence37) [inline ngshare_exchange/course_management.py:244]:
```suggestion
                    '{} was successfully added to {}'.format(
```

Comment (aalmanza1998) [reply ngshare_exchange/course_management.py]: fixed

**Pull request merged**